src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator, Draw {

```

Comment:
```suggestion
public class Circle extends Figure {
```
same for other child figures